app/javascript/packs/webauthn-setup.ts

```diff
@@ -66,11 +66,15 @@ function webauthn() {
           result.attestationObject;
         (document.getElementById('client_data_json') as HTMLInputElement).value =
           result.clientDataJSON;
-        (
-          document.getElementById('authenticator_data_value') as HTMLInputElement
-        ).value = `${result.authenticatorDataValue}`;
-        (document.getElementById('transports') as HTMLInputElement).value =
-          result.transports.join();
+        if (result.authenticatorDataFlagsValue) {
+          (
+            document.getElementById('authenticator_data_value') as HTMLInputElement
+          ).value = `${result.authenticatorDataFlagsValue}`;
+        }
+        if (result.transports) {
+          (document.getElementById('transports') as HTMLInputElement).value =
+            result.transports.join();
+        }
```

Comment:
The backend is already set up to accommodate that these values may be absent or otherwise non-valid:

https://github.com/18F/identity-idp/blob/492aec520124b3bfe480ef3c76dc6cbb23ff90bf/spec/forms/webauthn_setup_form_spec.rb#L202

https://github.com/18F/identity-idp/blob/492aec520124b3bfe480ef3c76dc6cbb23ff90bf/spec/forms/webauthn_setup_form_spec.rb#L113

Reply:
I also added a specific test case for missing `authenticator_data_value` in c86d787.